recipes/mumps_seq/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "mumps_seq" %}
```

Comment:
Mumps can be compiled for parallel or sequential use. The name `mumps_seq` stems from our `[homebrew](https://github.com/kwant-project/homebrew-kwant/blob/master/mumps_seq.rb)` recipe, because `mumps` already existed there.

I think it's therefore reasonable to change it back to `mumps`.